wallet/src/cli/programs/token.rs

```diff
@@ -49,6 +50,50 @@ pub enum TokenProgramAgnosticSubcommand {
         #[arg(long)]
         amount: u128,
     },
+    /// Burn tokens on `holder`, modify `definition`.
+    ///
+    /// `holder` is owned
+    ///
+    /// Also if `definition` is private then it is owned, because
+    /// we can not modify foreign accounts.
+    ///
+    /// ToDo: Return and add foreign variant when we could modify foreign accounts
```

Comment:
IMO this shouldn't be in user docs as clap will render it. Also let's have an issue